cmd/root.go

```diff
@@ -119,10 +120,19 @@ func rootCmd(o *options.Options) error {
 		return fmt.Errorf("GetEnabled: %w", err)
 	}
 
-	if o.Format == options.FormatDefault {
-		for checkName := range enabledChecks {
-			fmt.Fprintf(os.Stderr, "Starting [%s]\n", checkName)
+	// Define output to console or file
+	var output io.Writer = os.Stdout
+	if o.Output != "" {
+		outputF, err := os.Create(o.Output)
+		if err != nil {
+			return fmt.Errorf("unable to create output file: %w", err)
 		}
+		defer outputF.Close()
+		output = outputF
+	}
+
+	for checkName := range enabledChecks {
```

Comment:
this should remain gated behind `o.Format == options.FormatDefault`

Reply:
Is there a reason to only log this on `FormatDefault`? I thought that the information on which checks are running would still be relevant independent of outputting to the console or a file.

Reply:
If there was a reason, it wasn't well documented in the PR which added it: https://github.com/ossf/scorecard/pull/167